### ISSUE TITLE
Accessibility : Stats Graph Improvement 

### ISSUE THREAD

**Issue by jd-alexander** [wordpress-mobile/WordPress-Android#10987]
This graph was made accessible in a previous version of its implementation. More details #10285

## Expected behavior
Each bar in the graph should be accessible and when focused should be able to tell the user something. 

## Actual behavior
The whole view the graph resides in is announced. 

<img src="https://user-images.githubusercontent.com/1509205/71134051-dfcbb080-21ca-11ea-9b93-01aebd5dc7b0.png" width="320">

### Steps to reproduce the behavior
1. Go Stats within the app. 
2. Select either Day or Weeks and you will see the graph. 
3. Attempt to interact with the columns and you will see that's not possible. 

Tested on Google Pixel 4 XL, Android 10, WPAndroid `develop` ()
